package.json

```diff
@@ -12,11 +12,13 @@
     "@babel/helper-module-imports": "7.0.0",
     "@babel/preset-env": "7.3.1",
     "@babel/preset-typescript": "7.9.0",
+    "@lodder/grunt-postcss": "^3.1.1",
     "@types/webpack-env": "^1.15.1",
     "@typescript-eslint/eslint-plugin": "2.11.0",
     "@typescript-eslint/eslint-plugin-tslint": "2.11.0",
     "@typescript-eslint/parser": "2.11.0",
     "async": "2.6.0",
+    "autoprefixer": "^10.4.0",
```

Comment:
This used to be provided by `grunt-postcss` but due to postcss v8 changes we need to have it locally which seems correct in any case.